examples/data-sources/vmds_byoc/cloud_provider_regions/data-source.tf

```diff
@@ -0,0 +1,2 @@
+data "vmds_cloud_provider_regions" "all" {
+}
```

Comment:
i think the underlying API requires some params, check once

Reply:
Verified . It does not require any parameter .